src/screens/product-view-screen.tsx

```diff
@@ -22,7 +22,7 @@ export const ProductViewScreen: React.FC<routeID> = ({ route }) => {
     useEffect(() => {
         const getProducts = async () => {
             const docSnap = await getDoc(docRef);
-            const thing = docSnap.data();
+            const thing = docSnap.data() as DetailedProductData;
```

Comment:
We talked about this, but getting the type casting removed, figuring out how to do this later with firestore convertors down the line.